paloalto/package.yaml

```diff
@@ -0,0 +1,75 @@
+id: paloalto
+name: PaloAlto Networks
+author: Tenzir
+author_icon: https://raw.githubusercontent.com/tenzir/library/main/author.svg
+package_icon: https://raw.githubusercontent.com/tenzir/library/main/paloalto/package.svg
+
+description: |
+  [PaloAlto Networks](https://www.paloaltonetworks.com) solutions analyze and control network traffic by identifying applications and users, 
+  facilitating effective policy enforcement and threat prevention.
+  
+  This package simplifies the process of integrating and managing logs generated by Palo Alto Networks devices.
+
+inputs:
+  url:
+    name: URL
+    description: A URL where PaloAlto logs are provided. 
+    type: string
+
+pipelines:
+  onboard-from-url:
+    name: Onboard and parse PaloAlto Logs from URL
+    description: Onboards and parses PaloAlto Networks logs from a URL based on content type Traffic, Globalprotect, Threat, URL Filtering,
+      Data Filtering, Correlation, Tunnel Inspection, Config, System, HIP Match, IP-Tag, User-ID, Decryption, Authentication or else Unknown.
+    definition: |
+      // tql2
+      from "{{ inputs.url }}"" {
+      read_syslog
+      }
+      if ",TRAFFIC," in content {
+        legacy "parse content csv --header 'future_use1,receive_time,serial_number,type,log_subtype,version,generated_time,src_ip,dest_ip,src_nat_ip,dest_nat_ip,rule,src_user,dest_user,app_name,virtual_system,src_zone,dest_zone,interface_inbound,interface_outbound,log_action,future_use2,session_id,repeat_count,src_port,dest_port,src_nat_port,dest_nat_port,flags,protocol_name,status,bytes,bytes_out,bytes_in,packets,start_time,processed_time,panw_panos_url_category,future_use3,panw_panos_sequence_number,action_flags,src_location,dest_location,future_use4,packets_out,packets_in,session_end_reason,device_group_hierarchy_level1,device_group_hierarchy_level2,device_group_hierarchy_level3,device_group_hierarchy_level4,virtual_system_name,device_name,status_src,src_vm_uuid,dest_vm_uuid,tunnel_id_imsi,monitor_tag_imei,parent_session_id,parent_start_time,tunnel_type,sctp_association_id,sctp_chunks,sctp_chunks_sent,sctp_chunks_received,rule_uuid,http2_connection,app_flap_count,policy_id,link_switches,sd_wan_cluster,sd_wan_device_type,sd_wan_cluster_type,sd_wan_site,dynamic_user_group_name,xff_address,src_device_category,src_device_profile,src_device_model,src_device_vendor,src_device_os_family,src_device_os_version,src_hostname,src_mac,dest_device_category,dest_device_profile,dest_device_model,dest_device_vendor,dest_device_os_family,dest_device_os_version,dest_hostname,dest_mac,container_id,pod_namespace,pod_name,src_external_dynamic_list,dest_external_dynamic_list,host_id,serialnumber,src_dynamic_address_group,dest_dynamic_address_group,session_owner,high_resolution_timestamp,slice_service_type,slice_differentiator,app_subcategory,app_category,app_technology,app_risk,app_characteristic,app_container,tunneled_app,app_sass,app_sanctioned_state,offloaded,future_use5'"
```

Comment:
Should we drop all fields starting with `future_use` after parsing? 

```suggestion
        legacy "parse content csv --header 'future_use1,receive_time,serial_number,type,log_subtype,version,generated_time,src_ip,dest_ip,src_nat_ip,dest_nat_ip,rule,src_user,dest_user,app_name,virtual_system,src_zone,dest_zone,interface_inbound,interface_outbound,log_action,future_use2,session_id,repeat_count,src_port,dest_port,src_nat_port,dest_nat_port,flags,protocol_name,status,bytes,bytes_out,bytes_in,packets,start_time,processed_time,panw_panos_url_category,future_use3,panw_panos_sequence_number,action_flags,src_location,dest_location,future_use4,packets_out,packets_in,session_end_reason,device_group_hierarchy_level1,device_group_hierarchy_level2,device_group_hierarchy_level3,device_group_hierarchy_level4,virtual_system_name,device_name,status_src,src_vm_uuid,dest_vm_uuid,tunnel_id_imsi,monitor_tag_imei,parent_session_id,parent_start_time,tunnel_type,sctp_association_id,sctp_chunks,sctp_chunks_sent,sctp_chunks_received,rule_uuid,http2_connection,app_flap_count,policy_id,link_switches,sd_wan_cluster,sd_wan_device_type,sd_wan_cluster_type,sd_wan_site,dynamic_user_group_name,xff_address,src_device_category,src_device_profile,src_device_model,src_device_vendor,src_device_os_family,src_device_os_version,src_hostname,src_mac,dest_device_category,dest_device_profile,dest_device_model,dest_device_vendor,dest_device_os_family,dest_device_os_version,dest_hostname,dest_mac,container_id,pod_namespace,pod_name,src_external_dynamic_list,dest_external_dynamic_list,host_id,serialnumber,src_dynamic_address_group,dest_dynamic_address_group,session_owner,high_resolution_timestamp,slice_service_type,slice_differentiator,app_subcategory,app_category,app_technology,app_risk,app_characteristic,app_container,tunneled_app,app_sass,app_sanctioned_state,offloaded,future_use5'"
        content = content.drop_matching("^future_use")
```

Reply:
If we do this, then we should definitely explain this in the description. We could additionally consider assigning to `this` instead of `content` to drop the Syslog framing, which the competition appears to be doing partially.